packages/ui/src/utils/numberHelpers.ts

```diff
@@ -1,3 +1,4 @@
+import Big from "big.js";
```

Comment:
Is this something we should use instead of JSBI? I'd like to pick a single one if possible. I just picked JSBI because it was in some sample code, so I'm not tied to it, but it's used quite a bit.

Reply:
I'll make tickets to refactor existing JSBI into big.js at some point